src/lib/showToast.tsx

```diff
@@ -24,6 +37,20 @@ const showToast = (title: string, subtitle?: string, options?: ToastOptions) =>
       >
         {subtitle}
       </div>
+      {buttons ? (
+        <div
+          style={{
+            display: 'flex',
+            gap: '8px',
+          }}
+        >
+          {buttons.map(({ text, onClick }) => (
+            <Button key={text} onClick={onClick} size="small">
+              {text}
+            </Button>
+          ))}
```

Comment:
Will React not throw warnings about these `Button`s not having a `key`?

Reply:
They do have a key, don't they?
`key={text}`

Reply:
You're right, my eyes just skipped to the end of the line